### PR TITLE
goaccess 0.9.7

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by DomT4** [comment]
```
==> /usr/local/Cellar/goaccess/0.9.7/bin/goaccess --time-format=%T --date-format=%d/%b/%Y --log-format='%h %^[%d:%t %^] "%r" %s %b "%R" "%u"' -f access.log -o json 2>/dev/null
Error: goaccess: failed
uninitialized constant Formulary::FormulaNamespaced52f95cf026910bc6411cf9bbcdce3b0::Goaccess::JSON
```

File: Library/Formula/goaccess.rb [hunk 1]
```diff
@@ -39,7 +39,7 @@ def install
   end
 
   test do
-    require "json"
+    require "utils/json"
```
Comment (bfontaine) [inline Library/Formula/goaccess.rb]: Please replace `JSON` below with `Utils::JSON`.

**Comment by DomT4** [comment]
Still failing on Mavericks I'm afraid:
```
==> /usr/local/Cellar/goaccess/0.9.7/bin/goaccess --time-format=%T --date-format=%d/%b/%Y --log-format='%h %^[%d:%t %^] "%r" %s %b "%R" "%u"' -f access.log -o json 2>/dev/null
Error: goaccess: failed
undefined method `parse' for Utils::JSON:Module
```

**Comment by DomT4** [comment]
The Mavericks CI is using Ruby 1.8, so it's possible that method doesn't exist on Ruby 1.8.

**Comment by bfontaine** [comment]
It’s `Utils::JSON.load`.

**Comment by nijikon** [comment]
@bfontaine I know, it's already running build with that fix, thanks!

**Comment by nijikon** [comment]
Please merge, or maybe it's time that I would be granted access to merge myself? :dancer:

**Comment by bfontaine** [comment]
Merged! :stuck_out_tongue:

**Pull request closed without merging**